### ISSUE TITLE
Default dev port may be blocked by your network

### ISSUE THREAD

**Issue by Mazuh** [Mazuh/PosGraduacao#8]
Today I tried to run ```app.py``` in [IMD/UFRN](http://portal.imd.ufrn.br/) building, but turned out that all ports were blocked in this network but 80, even for my localhost. However, my machine doesn't allow me to access my own 80 without super user permission. So after having already done all the ```readme.md``` instructions, I opened a new terminal window, executed ```sudo su```, changed directory again to MinervaEnv and ran all activating processing, finishing with ```python app.py``` which output "0.0.0.0:80" (this will make the application available for all network too).

(Note that using ```sudo``` only **may** not work, since the operational system will try to run pip using its global modules instead of the current virtualenv modules.)

**Comment by Mazuh** [comment]
Wrong. The real cause is about to be fixed.
